packages/dd-trace/src/config.js

```diff
@@ -44,6 +44,14 @@ class Config {
       process.env.DD_LOGS_INJECTION,
       false
     )
+    const DD_OBFUSCATION_QUERY_STRING_REGEXP = coalesce(
+      options.qsObfuscator,
+      process.env.DD_OBFUSCATION_QUERY_STRING_REGEXP,
+      `(?:p(?:ass)?w(?:or)?d|pass(?:_?phrase)?|secret|(?:api_?|private_?|public_?|access_?|secret_?)key(?:_?id)?|token|\
```

Comment:
Can this be stored somewhere where it will be easier to diff?

Reply:
Any ideas ? I'm up for it but I honestly don't know where storing this would make it easier to diff.

Reply:
At a minimum, let's put it in a variable at the very top of the file and let's keep it on a single line with an eslint ignore.